tools/myst/enc/clock.c

```diff
@@ -72,7 +72,8 @@ static long _get_monotime()
     {
         // maintain monotonicity.
         // TODO: issue a warning. Host might be playing tricks.
-        return ++prev;
+        prev += 100;
```

Comment:
Can you add a comment here on why we are incrementing by 100.

Reply:
Thanks for pointing out. Added a comment

Reply:
The precision is still determined by how often the host thread updates the clock (which is every microsecond). But we are saying for two sub-microsecond consecutive clock_gettime queries, we spread them out 100ns apart, which is more realistic because even natively, a clock_gettime query takes the kernel 20-50 ns to handle.